Create Executive vote - August 10, 2022.md (#1903)
makerdao/community: governance/votes/Executive vote - August 10, 2022.md

### MOMC Parameter Changes

As per this [governance poll](https://vote.makerdao.com/polling/QmfMRfE4) on the latest MakerDAO Open Market Committee [proposal](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-28-july-2022/16895), the following parameter changes will be made if this executive proposal passes:

**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Reductions**

- Reduce **ETH-B** Stability Fee from 4% **to 3.75%**.
- Reduce the **WSTETH-A** Stability Fee from 2.50% to **2.25%**.
- Reduce the **WSTETH-B** Stability Fee from 0.75% to **0%**.
- Reduce the **WBTC-B** Stability Fee from 4.00% to **3.75%**.

**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Increases**

- Increase the **GUNIV3DAIUSDC1-A** Stability Fee from 0.01% to **0.02%**.
- Increase the **GUNIV3DAIUSDC2-A** Stability Fee from 0.05% to **0.06%**.
- Increase the **UNIV2DAIUSDC-A** Stability Fee from 0.01% to **0.02%**.

**[Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increases**

> - Increase the **GUNIV3DAIUSDC2-A** Maximum Debt Ceiling from 1 billion DAI to **1.25 billion DAI**.
> - Increase the **GUNIV3DAIUSDC1-A** Maximum Debt Ceiling from 750 million DAI to **1 billion DAI**.

Reviewer: I appreciate that this is my fault because I made the spreadsheet, but we've reversed the order of the GUNI vaults in the stab fee and line sections. Suggest making it consistent for readability reasons.

```suggestion
- Increase the **GUNIV3DAIUSDC1-A** Maximum Debt Ceiling from 750 million DAI to **1 billion DAI**.
- Increase the **GUNIV3DAIUSDC2-A** Maximum Debt Ceiling from 1 billion DAI to **1.25 billion DAI**.
```

- Increase the **MANA-A** Maximum Debt Ceiling from 15 million DAI to **17 million DAI**.

### Activate SocGen Vault

> The SocGenForge OFH token was [onboarded on August 2, 2022](https://vote.makerdao.com/executive/template-executive-vote-onboarding-real-world-asset-vaults-july-29-2022). However, due to the key holders being unavailable during the spell creation and validation, [the ownership of the `operator` wallet could not be verified](https://forum.makerdao.com/t/socgen-forge-ofh-granting-final-permission-after-the-onboarding/17033). 
> 
> [The ownership has now been verified](https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605/42). The final permission to the SocGen's wallet will be granted if this executive proposal passes.

Reviewer: ```suggestion
The SocGenForge OFH token was [onboarded on August 2, 2022](https://vote.makerdao.com/executive/template-executive-vote-onboarding-real-world-asset-vaults-july-29-2022). However, due to the key holders being unavailable during the spell creation and validation, the ownership of the `operator` wallet could not be verified. 

[The ownership has now been verified](https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605/42). The final permission to SocGen's wallet which will allow them to drawdown DAI will be granted, if this executive proposal passes.

Please see the [this post](https://forum.makerdao.com/t/socgen-forge-ofh-granting-final-permission-after-the-onboarding/17033) from the Collateral Engineering Services Core Unit (CES-001) that explains this in more detail.
```

summary: Huntingdon Valley (HVBank) Vault Drawdown; MOMC parameter changes, including Stability Fees and Maximum Debt Ceilings; SocGen Vault activation (OFH token, onboarded as RWA008-A); Recognized Delegates compensation (121,808 DAI).
date: 2022-08-10T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal]HVBank Vault Drawdown, MOMC Parameter Changes, SocGen Vault Activation, Recognized Delegate Compensation - August 10, 2022

The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.

---

## Executive Summary

If this executive proposal passes, the following **changes** will occur within the Maker Protocol:

> - **25,000,000 DAI** will be drawn down for HVBank, detailed below.

Reviewer: One word

```suggestion
- **25,000,000 DAI** will be drawndown for HVBank, as detailed below.
```

- SocGen Forge will be authorized to drawdown DAI from the previously activated SocGen Vault (RWA-008).
- A total of **121,808 DAI**, detailed below, will be transferred to multiple Recognized Delegates.

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

> This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.

Reviewer: Office hours isn't on for keeper reasons, reworded

```suggestion
This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that the recipients of the funds for HVBank are available at the time of execution of this executive proposal.
```
